websites/J/jpdb/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+		clientId: "988699263775154176",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let dueCountElem: Element, deckTitle: string, query: string, meaning: string;
+const decklist: string[] = [
+	"/novel",
+	"/vocabulary-list",
+	"/textbook",
+	"/anime",
+	"/youtube-video",
+	"/live-action",
+	"/video-game",
+	"/web-novel",
+	"/visual-novel",
+	"/audio",
+];
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "jpdb",
+		smallImageText: "jpdb.io",
+		startTimestamp: browsingTimestamp,
+	};
+	if (document.location.hostname === "jpdb.io") {
+		if (document.location.pathname.includes("/learn")) {
+			presenceData.details = "Viewing learn page";
+			dueCountElem =
+				document.querySelector('a[href="/learn"]').firstElementChild;
+			if (dueCountElem.getAttribute("style") === "color: green;")
```

Comment:
this seems unstable, is there no element or text you can check for instead?

Reply:
while there are text showing on `/learn` endpoint, I think it's more consistent to use navbar text considering that `/review` endpoint doesn't have any other text than the link on the navbar?
![image](https://user-images.githubusercontent.com/3771813/175362832-1dae52ca-45d8-49cd-ae6b-2eebce853394.png)